test/invoice_test.rb

```diff
@@ -19,4 +28,40 @@ def test_invoice_has_status
 
     assert_equal "shipped", invoice.status
   end
+
+  def test_find_transactions_returns_transactions_associated_with_invoice
+    result = @invoice_repository.invoices[0].transactions
+
+    assert_equal 1, result.count
+  end
+
+  def test_find_invoice_items_returns_invoice_items_associated_with_invoice
+    result = @invoice_repository.invoices[0].invoice_items
+
+    assert_equal 8, result.count
+  end
+
+  def test_find_customer_returns_customer_associated_with_invoice
+    result = @invoice_repository.invoices[0].customer
+
+    assert_equal "Joey", result.first_name
+  end
+
+  def test_find_merchant_returns_merchant_associated_with_invoice
+    result = @invoice_repository.invoices[9].merchant
+
+    assert_equal "Bechtelar, Jones and Stokes", result.name
+  end
+
+  def test_find_items_returns_items_associated_with_invoice
+    result = @invoice_repository.invoices[0].items
+
+    assert_equal 8, result.count
+  end
+
+  private
+
+  def fixtures_directory
+    File.expand_path('../../data/fixtures', __FILE__)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.